web/src/api/MercuryAPI.tsx

```diff
@@ -1,9 +1,10 @@
-import { Configuration, TodosApi } from './generated'
+import { ComponentApi, Configuration } from './generated'
 
-export class MercuryAPI extends TodosApi {
+// TODO: Combine all generated api's into MercuryAPI
+export class MercuryAPI extends ComponentApi {
   constructor(token: string) {
     const configuration = new Configuration({
-      basePath: 'http://localhost',
+      basePath: '',
```

Comment:
Maybe use a constant for basePath?

Reply:
There is no value. `http://localhost` has been removed.